script/lib/download-ffi.sh

```diff
@@ -17,7 +17,7 @@ if [[ $(find "${FFI_DIR}" -name "${FFI_VERSION}*") ]]; then
 fi
 
 warn "Cleaning ffi directory $FFI_DIR"
-rm -rf "${FFI_DIR:?}/*"
```

Comment:
There's a reason I did this, but I don't remember what it was. It might be that I didn't want to remove the folder in some cases. As long as it works this way, it's fine.

Also, now that I re-read this line it could be improved with an error message:

```
rm -rf "${FFI_DIR:?FFI_DIR was not set}"
```

Not super important, as the `:?` is only there as a safeguard if `set -u` is removed or didn't take for some reason.

Reply:
Update: Actually, this script should start:

```
#!/bin/bash -eu
set -e
set -u
```

because the github windows bash doesn't respect the shebang line.

Reply:
That was a gem of a tim's tech tid bit I found, as my windows scripts were unusually unflakey. they can't all pass first time right 😅 yep some errorz were being swallowed.

I checked to see if there was anything else in the folder we needed that might get trashed but it seemed not, I know deleting the whole standalone folder is a no no as it contains the `install.ts` script